ppr-ui/src/utils/feature-flags.ts

```diff
@@ -90,3 +90,15 @@ export async function updateLdUser (
 export function getFeatureFlag (name: string): any {
   return ldClient ? ldClient.variation(name, defaultFlagSet[name]) : defaultFlagSet[name]
 }
+
+/**
+ * Sets all boolean properties in the given default flag set object to true.
+ * @param defaultValue - The default flag value.
+ */
+export function setAllFlagDefaults(defaultValue: boolean): void {
```

Comment:
Nice, what a timing, just this week I was thinking the same, how each time the flags needs to reset and what would be a good solution. This is neat! 👍🏻

Reply:
When you mentioned that earlier i thought the same thing, this was overdue!
Saves us remembering the defaults constantly 👍